web/src/components/RunDetailAutomateMethods/methods/DeepLink/DeepLink.tsx

```diff
@@ -0,0 +1,37 @@
+import {FramedCodeBlock} from 'components/CodeBlock';
+import {Typography} from 'antd';
+// import {ReadOutlined} from '@ant-design/icons';
+// import {CLI_RUNNING_TESTS_URL} from 'constants/Common.constants';
```

Comment:
👀

Reply:
I believe this will be soon added, but I'm removing this code for the moment